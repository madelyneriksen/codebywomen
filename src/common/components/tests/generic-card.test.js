import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../generic-card';


describe("Navbar", () => {
  it("Matches Snapshot", () => {
    const tree = renderer.create(<Card />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})
