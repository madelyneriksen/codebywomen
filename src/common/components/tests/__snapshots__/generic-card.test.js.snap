// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generic Card Matches Snapshot 1`] = `
<div
  className="bg-white shadow-3 w-100 mv3 pa3"
/>
`;
